backend/tests/test_api_integration.py

```
@@ -0,0 +1,437 @@
+            files = {"file": ("test_audio.mp3", f, "audio/mpeg")}
+            data = {
+                "artist": "Test Artist",
+                "title": "Test Song"
+            }
+            
+            # Remove Content-Type header for multipart/form-data
+            headers = {
+                "Authorization": auth_headers["Authorization"]
+            }
+            
+            response = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)